src/BlazorUI/Bit.BlazorUI/Components/Notifications/Persona/BitPersona.scss

```
@@ -22,18 +22,42 @@
     --bit-prs-presence-glyph-sm: #{rem2(4px)};
     --bit-prs-presence-glyph-md: #{rem2(6.8px)};
     --bit-prs-initials-lg: #{rem2(42px)};
+    // How far the presence dot hangs off the coin. One knob, read by all four of the positions the dot
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)